index.html

```diff
@@ -16,29 +16,24 @@
     <div class="container">
       <header>
         <!-- Nav Section -->
-        <nav class="menu">
+        <nav class="navbar">
           <a href="#" class="logo">My Logo</a>
 
-          <ul>
-            <li>
-              <a href="#" class="active" onclick="markMenuItem(this)"
-                >Portfolio</a
-              >
-            </li>
-            <li><a href="#" onclick="markMenuItem(this)">About</a></li>
-            <li><a href="#" onclick="markMenuItem(this)">Contact</a></li>
+          <ul class="nav-menu">
+            <li class="nav-item"><a href="#portfolio" class="nav-link">Portfolio</a></li>
+            <li class="nav-item"><a href="#abouts" class="nav-link">About</a></li>
+            <li class="nav-item"><a href="#contacts" class="nav-link">Contact</a></li>
```

Comment:
- [ ] Let's kindly make sure that our menu options follow the design from the template(font, images, text, margins). In the screenshots below, you'll discover the bg is not transparent as in the Figma template.

| Yours | Figma |
| --- | --- |
| ![Screenshot from 2022-12-19 20-52-07](https://user-images.githubusercontent.com/63087491/208509644-7828d2a5-cbc6-4a2f-997e-ed44fadc2e89.png) | ![Screenshot from 2022-12-19 20-51-36](https://user-images.githubusercontent.com/63087491/208509688-60909b01-5508-472c-a029-d3d970c47b66.png) |